MMM-WeatherChart.js

```diff
@@ -118,17 +122,8 @@ Module.register("MMM-WeatherChart", {
         return params;
     },
 
-    getDayString: function (num) {
-        let dayString = [];
-        dayString.push('Su.');
-        dayString.push('Mo.');
-        dayString.push('Tu.');
-        dayString.push('We.');
-        dayString.push('Th.');
-        dayString.push('Fr.');
-        dayString.push('Sa.');
-
-        return dayString[num];
+    getDayString: function (dateTime) {
+        return dateTime.toLocaleString(moment.locale(), {weekday: 'short'}).substring(0, 2);
```

Comment:
Is it possible to enable to select the locale from configuration? It's more useful.

For example:
```
dateTime.toLocaleString(moment.locale(this.config.locale), {weekday: 'short'}).substring(0, 2);
```

Reply:
That's like it's done with other modules, they take the locale from the global configuration, not their own.
Adding an extra locale to the weatherchart config would be a bit odd.
But the default could be "", then the global is used. Otherwise the specific will be used.

Reply:
I found the language config in general config as you said.

https://docs.magicmirror.builders/getting-started/configuration.html#general

Thanks.